### PR TITLE
Add pagination type to shortcode params

### LINKED ISSUE #5977: Set pagination type in shortcode
https://wordpress.org/support/topic/pagination-type-via-pods-shortcode/

Optionally set the pagination type in the `pagination` shortcode param.

### PR DESCRIPTION
**Pull request by JoryHogeveen**
## Description

~~Optionally parse the `pagination` parameter as the pagination type.~~
Added `pagination_type` param.

## Related GitHub issue(s)

Fixes #5977

## PR checklist

- [ ] I have tested my own code to confirm it works as I intended.
- [x] My code follows the [WordPress Coding Standards](https://make.wordpress.org/core/handbook/best-practices/coding-standards/).
- [x] My code follows the [WordPress Inline Documentation Standards](https://make.wordpress.org/core/handbook/best-practices/inline-documentation-standards/).

### CODE REVIEW

File: includes/general.php [hunk 1]
```diff
@@ -1108,19 +1109,24 @@ function pods_shortcode_run( $tags, $content = null ) {
 		return $return;
 	}//end if
 
+	$pagination = false;
+	if ( ! $is_singular && 0 < $found && $tags['pagination'] ) {
```
Comment (sc0ttkclark) [inline includes/general.php:1113]:
```suggestion

	// Only handle pagination on non-singular shortcodes where items were found.
	if ( ! $is_singular && 0 < $found && $tags['pagination'] ) {
```

File: includes/general.php [hunk 2]
```diff
@@ -1108,19 +1109,24 @@ function pods_shortcode_run( $tags, $content = null ) {
...
 	}
 
-	if ( ! $is_singular && 0 < $found && true === $tags['pagination'] && in_array(
-		$tags['pagination_location'], array(
-			'before',
-			'both',
-		), true
-	) ) {
-		echo $pod->pagination( $tags['pagination_label'] );
+	if ( $tags['pagination'] && in_array( $tags['pagination_location'], array( 'before', 'both', ), true ) ) {
```
Comment (sc0ttkclark) [inline includes/general.php:1128]:
```suggestion
	if ( $pagination && in_array( $tags['pagination_location'], array( 'before', 'both', ), true ) ) {
```

File: includes/general.php [hunk 3]
```diff
@@ -1132,13 +1138,8 @@ function pods_shortcode_run( $tags, $content = null ) {
...
 	echo $content;
 
-	if ( ! $is_singular && 0 < $found && true === $tags['pagination'] && in_array(
-		$tags['pagination_location'], array(
-			'after',
-			'both',
-		), true
-	) ) {
-		echo $pod->pagination( $tags['pagination_label'] );
+	if ( $tags['pagination'] && in_array( $tags['pagination_location'], array( 'after', 'both', ), true ) ) {
```
Comment (sc0ttkclark) [inline includes/general.php:1141]:
```suggestion
	if ( $pagination && in_array( $tags['pagination_location'], array( 'after', 'both', ), true ) ) {
```

**Review by sc0ttkclark — CHANGES_REQUESTED** [review]
Minor tweaks needed

File: includes/general.php (hunk 2 above)
Comment (JoryHogeveen) [reply includes/general.php:1128]: This is invalid, `$pagination` could also be an empty array.

File: includes/general.php (hunk 3 above)
Comment (JoryHogeveen) [reply includes/general.php:1141]: This is invalid, `$pagination` could also be an empty array.

File: includes/general.php [hunk 4]
```diff
@@ -1108,19 +1109,26 @@ function pods_shortcode_run( $tags, $content = null ) {
...
 
+	$pagination = false;
+
+	// Only handle pagination on non-singular shortcodes where items were found.
+	if ( ! $is_singular && 0 < $found && $tags['pagination'] ) {
+		$pagination = array(
+			'label' => pods_v( 'pagination_label', $tags, null ),
+			'type'  => pods_v( 'pagination_type', $tags, null ),
+		);
+		// Remove empty params.
```
Comment (sc0ttkclark) [inline includes/general.php:1120]:
```suggestion

		// Remove empty params.
```

**Review by sc0ttkclark — DISMISSED** [review]

**Pull request merged**